tests/routers/test_chat_stream.py

```
@@ -0,0 +1,289 @@
+            assert chat_response.status_code == 200
+        
+        # 5. 메시지 히스토리 다시 조회 (메시지 추가됨)
+        final_messages_response = await async_client.get(
+            f"/chat/sessions/{session_id}/messages",
+            params={"user_id": user_id}
+        )
+        assert final_messages_response.status_code == 200
+        final_messages_data = final_messages_response.json()
+        
+        # 사용자 메시지와 AI 응답이 저장되었는지 확인
+        assert len(final_messages_data["messages"]) >= 1 
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)